### DISCUSSION TITLE
Fluid BIC

### DISCUSSION THREAD

**Discussion by Buddh1n1** [iotaledger/iota-core#706] [Ideas]
For better UX we could make mana more fluid - in particular the BIC.
Users can transfer mana from one BIC entry to another with an onchain request that get's executed after the slot is committed/finalized.

**Comment by alexsporn** [comment]
This does not fit the current UTXO model and thus brings a whole new can of worms regarding double spends and conflict resolution.

**Comment by Buddh1n1** [comment]
I see, thanks
